dropwizard-migrations/src/main/java/io/dropwizard/migrations/CloseableLiquibaseWithFileSystemMigrationsFile.java

```diff
@@ -17,9 +21,25 @@ public class CloseableLiquibaseWithFileSystemMigrationsFile extends CloseableLiq
         String file
     ) throws LiquibaseException, SQLException {
         super(file,
-              new FileSystemResourceAccessor(),
-              database,
-              dataSource);
+            new FileSystemResourceAccessor(getRootPaths().toArray(new File[0])),
+            database,
+            dataSource);
+    }
+
+    private static List<File> getRootPaths() {
+        List<File> rootPaths = new ArrayList<>();
+        boolean isWindows = System.getProperty("os.name", "").toLowerCase().contains("win");
+        if (isWindows) {
+            rootPaths.add(new File("C:\\"));
+        } else {
+            rootPaths.add(new File("/"));
+        }
```

Comment:
Should we let Java NIO do the hard work and do something like this?
```suggestion
        rootPaths.add(Paths.get("/").toFile());
```

https://docs.oracle.com/javase/tutorial/essential/io/pathOps.html#create

Reply:
Thanks for the suggestion. But in the linked docs I can't read that using `/` would work on Windows too.

After looking deeper into the docs about that, I would change the code to use [`FileSystems.getDefault().getRootDirectories()`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/FileSystem.html#getRootDirectories()). This way we would avoid to deal with OS specific paths and would additionally cover situations with multiple drives.

Reply:
Yes, let's use `getRootDirectories()`. 👍